src/ptc/util/jms.clj

```diff
@@ -24,40 +24,65 @@
    :uuid (str uuid)
    :notifications []})
 
-(def params-keys->jms-keys
-  "Map params.txt keys to their associated keys in a JMS message."
-  {:CHIP_TYPE_NAME         :chipName
-   :CHIP_WELL_BARCODE      :chipWellBarcode
-   :INDIVIDUAL_ALIAS       :collaboratorParticipantId
-   :LAB_BATCH              :labBatch
-   :PARTICIPANT_ID         :participantId
-   :PRODUCT_FAMILY         :productFamily
-   :PRODUCT_NAME           :productName
-   :PRODUCT_ORDER_ID       :productOrderId
-   :PRODUCT_PART_NUMBER    :productPartNumber
-   :REGULATORY_DESIGNATION :regulatoryDesignation
-   :RESEARCH_PROJECT_ID    :researchProjectId
-   :SAMPLE_ALIAS           :collaboratorSampleId
-   :SAMPLE_GENDER          :gender
-   :SAMPLE_ID              :sampleId
-   :SAMPLE_LSID            :sampleLsid})
-
-(def notification-keys->jms-keys-table
+(def wfl-keys->jms-keys-table
   "How to satisfy notification keys in WFL request."
-  ["action" "request notification key"   "JMS key"
-   ::copy   :analysis_version_number     :analysisCloudVersion
-   ::copy   :chip_well_barcode           :chipWellBarcode
-   ::copy   :reported_gender             :gender
-   ::copy   :sample_alias                :sampleAlias
-   ::copy   :sample_lsid                 :sampleLsid
-   ::copy   :call_rate_threshold         :callRateThreshold
-   ::chip   :bead_pool_manifest_file     :beadPoolManifestPath
-   ::chip   :cluster_file                :clusterFilePath
-   ::chip   :extended_chip_manifest_file :chipManifestPath
-   ::chip   :gender_cluster_file         :genderClusterFilePath
-   ::chip   :zcall_thresholds_file       :zCallThresholdsPath
-   ::push   :green_idat_cloud_path       :greenIDatPath
-   ::push   :red_idat_cloud_path         :redIDatPath])
+  ["action" "req'd?" "request notification key"   "JMS key"
```

Comment:
Now that we also makes `control_***` inputs optional [here](https://github.com/broadinstitute/wfl/blob/6234cbd6af7dde5d79a5e7c4be781538580aa9aa/api/src/zero/module/aou.clj#L76), does it make sense to add them here as optional keys?

Reply:
(I remember @pshapiro4broad mentioned the aou samples won't have control params, so this is totally optional if that's true)

Reply:
Maybe.  The only one I see in any of the messages is `controlDataDirectory`.
Not even sure what that is …

Reply:
Looks like `control_data_directory` is not a workflow input now, optional or otherwise.